update adding app: add a step with team selection (#597)
codemagic-ci-cd/codemagic-docs: content/getting-started/gitlab.md

Adding apps from GitLab requires granting access to your GitLab account to enable the integration and load repositories.

1. Click **Add application** in the top right corner of the Applications page.

> 2. **Select a team** where to add an application. By default, the application will be added to your personal account. You can add an application only to a team with enabled billing. This step is skipped if you don’t have teams where you’re an owner, and application will be added to your personal account.

Reviewer: ```suggestion
2. If you are an owner of a team in Codemagic, you'll be asked to **Select a team**. You can only add the application to a team that has billing enabled. If you don't own any teams, this step is skipped and the application will be added to your personal account.
```
